src/RunGui.cpp

```diff
@@ -145,17 +145,18 @@ namespace {
     // improved in the future.
     void clearCache(const QDir &dir)
     {
-        qCDebug(chatterinoCache) << "[Cache] cleared cache";
-
-        QStringList toBeRemoved;
-
+        int deletedCount = 0;
         for (auto &&info : dir.entryInfoList(QDir::Files))
         {
             if (info.lastModified().addDays(14) < QDateTime::currentDateTime())
             {
-                toBeRemoved << info.absoluteFilePath();
+                bool res = QFile(info.absoluteFilePath()).remove();
+                if (res)
+                    ++deletedCount;
             }
         }
+        qCDebug(chatterinoCache)
+            << "[Cache] deleted" << deletedCount << "files";
```

Comment:
`[Cache]` is redundant with categorized logging (`qCDebug(chatterinoCache)`)
```suggestion
            << "Deleted" << deletedCount << "files";
```